### PR TITLE
Fix Xwing tests using fandom url

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java [hunk 1]
```diff
@@ -147,6 +145,15 @@ public String getWikiGlobalURL(){
     }
   }
 
+  public String getFandomUrl() {
+    EnvType env = Configuration.getEnvType(EnvType.PROD.getKey());
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java]: MAJOR Rename "env" which hides the field declared at line 14. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AHiddenFieldCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 5 major

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L99](https://github.com/Wikia/selenium-tests/blob/60ec1aa3793acd50d143c4f4d28368cc98abd836/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L99): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L109](https://github.com/Wikia/selenium-tests/blob/60ec1aa3793acd50d143c4f4d28368cc98abd836/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L109): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L136](https://github.com/Wikia/selenium-tests/blob/60ec1aa3793acd50d143c4f4d28368cc98abd836/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L136): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L136](https://github.com/Wikia/selenium-tests/blob/60ec1aa3793acd50d143c4f4d28368cc98abd836/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L136): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L99](https://github.com/Wikia/selenium-tests/blob/e4546f580e2ecaeab788e7ad372847effb730e67/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L99): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L109](https://github.com/Wikia/selenium-tests/blob/e4546f580e2ecaeab788e7ad372847effb730e67/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L109): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L136](https://github.com/Wikia/selenium-tests/blob/e4546f580e2ecaeab788e7ad372847effb730e67/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L136): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L136](https://github.com/Wikia/selenium-tests/blob/e4546f580e2ecaeab788e7ad372847effb730e67/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L136): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java [hunk 2]
```diff
@@ -56,19 +55,18 @@ public String getUrlForPage(Page page, String qs) {
   }
 
   public String getUrlForPath(String wikiName, String wikiPath) {
-    String url = "";
-    if (!wikiPath.startsWith("/")) {
-      url = String.format("%s/%s", getUrlForWiki(wikiName), wikiPath);
-    } else {
-      url = String.format("%s%s", getUrlForWiki(wikiName), wikiPath);
-    }
+    return addPathToUrl(getUrlForWiki(wikiName), wikiPath);
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java]: 👍

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L99](https://github.com/Wikia/selenium-tests/blob/cbfea073821edd027e5520198485647d37b14c96/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L99): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L109](https://github.com/Wikia/selenium-tests/blob/cbfea073821edd027e5520198485647d37b14c96/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L109): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L136](https://github.com/Wikia/selenium-tests/blob/cbfea073821edd027e5520198485647d37b14c96/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L136): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L136](https://github.com/Wikia/selenium-tests/blob/cbfea073821edd027e5520198485647d37b14c96/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L136): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**